mod_wasm/modules/wasm/mod_wasm.c

```diff
@@ -282,18 +283,26 @@ static int content_handler(request_rec *r)
                 "content_handler() - ERROR! Couldn't set HTTP Request Body as stdin!");
       }
     }
-    /* run Wasm module */
-    const char* module_response = wasm_executionctx_run(exec_ctx_id);
+
+    /* run Wasm execution context */
+    size_t len = 0;
+    const char* module_response = NULL;
+    ret = wasm_executionctx_run(exec_ctx_id, &module_response, &len);
+    if ( ret != OK ) {
+        ap_log_rerror(APLOG_MARK, APLOG_ERR, ret, r,
+            "content_handler() - ERROR! Couldn't run Wasm execution context '%s'!", exec_ctx_id);
+    }
 
     if (dcfg->bWasmEnableCGI) {
       /*
        * Retrieve the CGI variables and feed our own response with
        * them; write the response from the module as our own response;
        * which has the headers already stripped from it.
        */
+      char buffer[MAX_STRING_LEN] = {'\0'};
       const char *termch;
       int termarg;
-      int ret = ap_scan_script_header_err_strs(r, NULL, &termch, &termarg, module_response, NULL);
+      ret = ap_scan_script_header_err_strs(r, buffer, &termch, &termarg, module_response, NULL);
```

Comment:
I see that it is common for `ap_scan_script_header_err_strs` to get a `NULL` on its second argument on Apache modules in the wild.

I don't know what specific errors `buffer` might contain, but if they are very specific, or if they contain verbatim request/response details, we might be leaking information of the request/response on the Apache logs.

I'd follow the convention that other widely used modules have established.